content/get-started/installation/_index.md

```diff
@@ -9,6 +9,20 @@ There are multiple ways to set up your development environment and install Strea
 Developing locally with Python installed on your own computer is the most
 common scenario.
 
+## Sample Streamlit without installing it
+
+Try a version of Streamlit that runs in your browser&mdash;no installation required.
+(This is not how Streamlit is meant to be used, because it has many downsides. That's why it's a
+_playground_!)
```

Comment:
I know this is heavily borrowed from my own text, but how about this change:

Try a Streamlit Playground that runs in your browser&mdash;no installation required. (Note that this is not how Streamlit is meant to be used, because it has many downsides. That's why it's a
_playground_!)